### ISSUE TITLE
Please continue to maintain, thank you !

### ISSUE THREAD

**Issue by fengyuanhui** [Ziver/Netty-Transport-jSerialComm#4]
![image](https://user-images.githubusercontent.com/16237143/89026328-1a99e780-d35b-11ea-8321-7c98b9afdc97.png)

**Comment by Ziver** [comment]
Are you having issues with the lib? could you please be a little bit more specific what you are trying to do and if possible maybe a small example code of the issue?

**Comment by fengyuanhui** [comment]
Yes, I use the latest version of jSerialComm in my project, version 2.6.2. Try to run "Netty - Transport -jSerialComm/SRC/test/Java/IO/Netty/example/jSerialComm JSerialCommClient. Java". And then there's the error above.Looking at the jSerialComm used in Netty-transport-JserialComm is 1.3.11.I suspect version incompatibilities are to blame.Can you upgrade jSerialComm in Netty-transport-JserialComm to the latest version and maintain it for the long term?
